static/js/main.js

```diff
@@ -1,48 +1,42 @@
-document.addEventListener("DOMContentLoaded", function(){
-  var toggle = document.getElementById("scheme-toggle");
-
-  var scheme = "light";
-  var savedScheme = localStorage.getItem("scheme");
-
-  var container = document.getElementsByTagName("html")[0];
-  var prefersDark = window.matchMedia("(prefers-color-scheme: dark)").matches;
-
-  if (prefersDark) {
-    scheme = "dark";
-  }
+function toggleDarkMode() {
+    var theme = localStorage.getItem('scheme');
+    var toggle = document.getElementById('scheme-toggle');
```

Comment:
This element does not exist by default and will `NullPointer` itself if `site.Params.mode` is unset.

Make sure to set 
```toml
  # Site color - dark/light/auto
  mode = "auto"
```

to the `exampeSite/config.toml`

Reply:
Two options, add a check inside `document.addEventListener` for it, or add it inside the function.

Reply:
We should just default the `exampleSite`, so it's clear to users.